### PR TITLE
travis.yml : Add command to run flake8

### PR DESCRIPTION
**Pull request by kriti21**
### Description
Flake8 is added in requirements.txt 
This adds command to run flake8 in .travis.yml

Fixes [#478](https://github.com/systers/vms/issues/478)

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ before_script:
   - export DJANGO_SECRET_KEY=foobarbaz
 script:
   - cd vms
+  - flake8 --show-source
```
Comment (tapasweni-pathak) [inline .travis.yml]: `flake8 .` would do. We can only add it once we also fix the flake8 issues with the code.

File: vms/event/services.py [hunk 2]
```diff
@@ -120,6 +120,13 @@ def get_events_by_date(start_date, end_date):
 
     return result
 
+def get_events_by_location(event_list, location):
```
Comment (tapasweni-pathak) [inline vms/event/services.py]: you may want to open a separate pr for this.

Comment (kriti21) [reply vms/event/services.py]: I am sorry. This is a commit mistake i think. I have made a separate PR for this.

File: .travis.yml (hunk 1 above)
Comment (kriti21) [reply .travis.yml]: So should I change it or leave it for later?

File: vms/event/services.py (hunk 2 above)
Comment (tapasweni-pathak) [reply vms/event/services.py]: O no problem! Just when you are submitting a change, good to club related into same pull request with separate commits, but these are completely unrelated.

File: .travis.yml (hunk 1 above)
Comment (tapasweni-pathak) [reply .travis.yml]: We can merge the `flake8 .` change once we also fix the issues that comes when you run this command in your local on vms. I hope this helps.

File: vms/event/services.py (hunk 2 above)
Comment (kriti21) [reply vms/event/services.py]: I will keep that in mind. Thank you :)

File: .travis.yml (hunk 1 above)
Comment (kriti21) [reply .travis.yml]: Okay. Got it. Thanks.

**Comment by ayushgarg1804** [comment]
@kriti21 Are you still working on this or can it be closed for another member to solve?

**Comment by ayushgarg1804** [comment]
Closing due to inactivity

**Pull request closed without merging**